components/brave_rewards/resources/extension/brave_rewards/manifest.json

```diff
@@ -55,6 +56,14 @@
       ]
     }
   ],
+  "externally_connectable": {
+    "ids": [
+      "*"
+    ],
+    "matches": [
+      "https://www.youtube.com/*"
```

Comment:
according to https://developer.chrome.com/extensions/manifest/externally_connectable, this doesn't affect content scripts but it does allow the youtube webpage to connect to our extension. what are we doing that requires the webpage itself to connect?

Reply:
I think you're right, it looks like it may not be necessary. I added it when I was trying to get two-way communication to work between the Brave Rewards extension and the Greaselion content script. It seemed to be necessary when having the content script connect to the Rewards extension. I'll remove it and verify that it still works. Will update with results here either way.

Reply:
It seems to work correctly without this, so I removed it. Thanks for mentioning it.